### ISSUE TITLE
Don't expose service names when sending emails with AWS SES

### ISSUE THREAD

**Issue by philherbert** [department-of-veterans-affairs/notification-api#140]
Currently, the email address that emails are sent from is constructed using:
- the service name
- the service `email_from` (or default `email_from` after #123)
- the service `sending_domain` or default `NOTIFY_EMAIL_DOMAIN`

```
from_address = '"{}" <{}@{}>'.format(service.name, service.email_from, sending_domain)
```

For email providers other than Govdelivery (eg AWS SES), the service name will appear as the email sender. (Govdelivery will just remove it and send from the email address, with no name)

Given that our service names are things like `vets-api` and `VEText`, we don't want those being shown to email recipients. Instead, emails should always come from `U.S. Department of Veterans Affairs`.

---

# Value Statement:
**As** a consuming service of VANotify
**I want to** send emails without my service name appearing as the sender
**So that** recipients receive emails from a consistent sender

# Acceptance Criteria
**GIVEN** AWS SES is the email provider of choice
**WHEN** I make a request to send an email notification
**THEN** the email sender, as visible to the recipient, is `U.S. Department of Veterans Affairs`

# Checklist:
- [x] Add a global configuration for something like `NOTIFY_EMAIL_FROM_NAME`
- [x] Use that configuration value when constructing the `from_address` for outgoing emails, instead of using the service name
- [x] Check this works on AWS SES

# Assumptions:
-

# Additional Info/Resources:
-

# Out of Scope:
-

# Open Questions:
-

**Comment by miabecker** [comment]
Priority will depend on SES speed.
